### PR TITLE
feat(wizard): add tokens and update shim

### PR DESCRIPTION
**Pull request by dtsanevmw**
Depends on #836 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-789

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by mindarelus — APPROVED** [review]
lgtm

File: projects/angular/src/wizard/_properties.wizard.scss [hunk 1]
```diff
@@ -25,7 +27,47 @@
       --clr-wizard-stepnav-active-border-radius: var(--clr-wizard-border-radius);
 
       --clr-wizard-stepnav-border-color: hsl(0, 0%, 89%);
-      --clr-wizard-box-shadow-color: rgba(0, 0, 0, 0.2);
+      --clr-wizard-box-shadow: 0
+        #{variables.$clr_baselineRem_1px}
+        #{variables.$clr_baselineRem_2px}
+        #{variables.$clr_baselineRem_2px}
+        rgba(0, 0, 0, 0.2);
```
Comment (valentin-mladenov) [inline projects/angular/src/wizard/_properties.wizard.scss:34]: Reintroduce `--clr-wizard-box-shadow-color: rgba(0, 0, 0, 0.2);` some app might overloading it.

```suggestion
        var(--clr-wizard-box-shadow-color);
```

File: projects/angular/src/wizard/_wizard.clarity.scss [hunk 2]
```diff
@@ -246,14 +281,27 @@
...
+        variables.$clr-use-custom-properties
+      );
+
       margin: 0 0 (-1 * variables.$clr-global-borderwidth) 0;
-      padding: variables.$clr_baselineRem_0_25 0;
-      padding-left: variables.$clr_baselineRem_8px;
+      @include mixins.css-var(
+        padding,
+        clr-wizard-stepnav-item-padding,
+        variables.$clr_baselineRem_0_25 0 variables.$clr_baselineRem_0_25 variables.$clr_baselineRem_4px,
```
Comment (valentin-mladenov) [inline projects/angular/src/wizard/_wizard.clarity.scss:302]: Original  padding left is 8px `variables.$clr_baselineRem_8px;` not 4.

```suggestion
        variables.$clr_baselineRem_0_25 0 variables.$clr_baselineRem_0_25 variables.$clr_baselineRem_8px,
```

File: projects/angular/src/wizard/_wizard.clarity.scss [hunk 3]
```diff
@@ -267,6 +315,21 @@
         variables.$clr-use-custom-properties
       );
 
+      &:not(.disabled):hover .clr-wizard-stepnav-link {
+        @include mixins.css-var(
+          color,
+          string.unquote('clr-wizard-stepnav-text--active'),
```
Comment (valentin-mladenov) [inline projects/angular/src/wizard/_wizard.clarity.scss:321]: I know and understand why have to be like that, but we have to rename this property after token migration.

File: projects/angular/src/wizard/_wizard.clarity.scss [hunk 4]
```diff
@@ -336,9 +405,20 @@
...
+      );
       font-size: variables.$clr_baselineRem_0_583;
       font-weight: inherit;
+      @include mixins.css-var(
+        line-height,
+        clr-wizard-stepnav-link-line-height,
+        variables.$clr_baselineRem_1,
+        variables.$clr-use-custom-properties
+      );
       letter-spacing: normal;
```
Comment (valentin-mladenov) [inline projects/angular/src/wizard/_wizard.clarity.scss:422]: Introduce topography changes.

File: projects/ui/src/shim.cds-core.scss [hunk 5]
```diff
@@ -952,21 +953,40 @@
   --clr-tooltip-arrow-vertical-size: calc(-1 * var(--cds-global-space-6));
 
   /* Wizard */
+  --clr-wizard-main-padding: var(--cds-global-space-9);
+  --clr-wizard-stepnav-link-line-height: calc(16 * (1rem / var(--clr-global-font-size)));
```
Comment (valentin-mladenov) [inline projects/ui/src/shim.cds-core.scss:957]: In specs step nav link's line height is 20.

**Review by valentin-mladenov — COMMENTED** [review]
Check my comments.

File: projects/angular/src/wizard/_properties.wizard.scss (hunk 1 above)
Comment (dtsanevmw) [reply projects/angular/src/wizard/_properties.wizard.scss:34]: Thanks for catching. I added `clr-wizard-box-shadow` with the idea to use the color too but missed it on the way.

File: projects/angular/src/wizard/_wizard.clarity.scss (hunk 2 above)
Comment (dtsanevmw) [reply projects/angular/src/wizard/_wizard.clarity.scss:302]: Originally the colored left "border" was made up with shadow and additional padding was needed. I change it to use border instead and that much padding is no longer needed.

File: projects/angular/src/wizard/_wizard.clarity.scss (hunk 3 above)
Comment (dtsanevmw) [reply projects/angular/src/wizard/_wizard.clarity.scss:321]: Yep, totally agree :)

**Pull request merged**